src/Sinch/Verification/Report/Request/FlashCallVerificationReportRequest.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace Sinch.Verification.Report
+{
+    public class FlashCallVerificationReportRequest : IVerifyReportRequest
+    {
+        /// <inheritdoc />
+        public override string Method { get; } = "flashCall";
```

Comment:
Is the value always be `flashCall` for a `FlashCallVerificationReportRequest`. I see it as an example in the spec

Reply:
Yes, it will always be the same, see: https://developers.sinch.com/docs/verification/api-reference/verification/tag/Verifications/#tag/Verifications/operation/ReportVerificationByIdentity!c=200&path=0/method&t=response

Reply:
Ok, I understand now that you have derived the VerifyReportRequest class for all kinds of requests 👍